pkg/gadgets/seccomp/gadget.go

```diff
@@ -173,12 +173,16 @@ func seccompProfileAddLabelsAndAnnotations(
 	trace *gadgetv1alpha1.Trace,
 	podName string,
 	containerName string,
+	ownerReference map[string]string,
 ) {
 	traceName := fmt.Sprintf("%s/%s", trace.ObjectMeta.Namespace, trace.ObjectMeta.Name)
 	r.ObjectMeta.Annotations["seccomp.gadget.kinvolk.io/trace"] = traceName
 	r.ObjectMeta.Annotations["seccomp.gadget.kinvolk.io/node"] = trace.Spec.Node
 	r.ObjectMeta.Annotations["seccomp.gadget.kinvolk.io/pod"] = podName
 	r.ObjectMeta.Annotations["seccomp.gadget.kinvolk.io/container"] = containerName
+	for key, value := range ownerReference {
+		r.ObjectMeta.Annotations["seccomp.gadget.kinvolk.io/ownerReference"+key] = value
```

Comment:
I think it would unfortunately not be a valid annotation: only one slash is valid:

https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/#syntax-and-character-set

Reply:
I confirm it cannot have more than one `/`. At most, we can do `seccomp.gadget.kinvolk.io/ownerReference-<Field>`. WDYT?

Reply:
> only one slash is valid:

ops. 

> I confirm it cannot have more than one /. At most, we can do seccomp.gadget.kinvolk.io/ownerReference-<Field>. WDYT?

yes, that's fine then.

Reply:
Done